src/metabase-report-notifier/src/main/java/com/champtitles/metabasereportexecutor/notifier/App.java

```diff
@@ -2,15 +2,114 @@
 
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.RequestHandler;
-import com.amazonaws.services.lambda.runtime.events.S3Event;
+import com.amazonaws.services.lambda.runtime.events.SNSEvent;
+import com.champtitles.metabasereportexecutor.executor.KmsDecrypt;
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-public class App implements RequestHandler<S3Event, Void> {
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.NoSuchElementException;
+
+public class App implements RequestHandler<SNSEvent, Void> {
+
+    private static final Logger logger = LoggerFactory.getLogger(App.class.getName());
```

Comment:
A static final variable is a constant and should follow constant naming convention (UPPER_SNAKE_CASE) by Java standard.